packages/react-components/src/templates/__tests__/EventPage.test.tsx

```diff
@@ -95,6 +93,18 @@ it('renders additional materials', () => {
   expect(getByText('Example Material')).toBeVisible();
 });
 
+it('renders continue the event conversation when group with slack provided', () => {
+  const { queryByText, rerender } = render(
+    <EventPage
+      {...props}
+      group={{ ...createGroupResponse(), tools: { slack: 'http://slack.com' } }}
+    />,
+  );
+  expect(queryByText(/continue the conversation/i)).toBeVisible();
+  rerender(<EventPage {...props} group={undefined} />);
+  expect(queryByText(/continue the conversation/i)).not.toBeInTheDocument();
```

Comment:
```suggestion
  expect(queryByText(/slack/i)).not.toBeInTheDocument();
```

just because it's more intuitive that something about slack will def have to be there if we give it a `slack` property, to know the headline about continuing the conversation you'd have to know the `EventConversation` component